source/SIL.AppBuilder.Portal.Frontend/src/data/push-payload.ts

```diff
@@ -2,20 +2,22 @@ import { schema, keyMap } from './schema';
 import { serializer } from './store';
 import { recordIdentityFrom } from './store-helpers';
 
-export async function pushPayload(updateStore, payload) {
+export async function pushPayload(updateStore, payload, op = 'addRecord') {
```

Comment:
can you make a PBI to to make this `op` arg not a freeform string, but a union type of specific strings? (or an enum?)